package-lock.json

```diff
@@ -0,0 +1,3 @@
+{
+  "lockfileVersion": 1
```

Comment:
If we indeed want to completely move to package-lock.json for a reproducible build, we should go with lockfileVersion 2 as it is also backwards compatible with older node cli versions that could work with only lockfileVersion 1.

Reply:
Removed this file:)